sessionctx/variable/sysvar.go

```diff
@@ -1715,6 +1715,9 @@ var defaultSysVars = []*SysVar{
 		return nil
 	}},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBRcWriteCheckTs, Type: TypeBool, Value: BoolToOnOff(DefTiDBRcWriteCheckTs), SetSession: func(s *SessionVars, val string) error {
+		if !s.RcWriteCheckTS {
+			metrics.RcWriteCheckTsSetCount.Inc()
+		}
 		s.RcWriteCheckTS = TiDBOptOn(val)
```

Comment:
Should it be placed after the value is set?
The global variable value of `rc_read_check_ts` is recorded for telemetry. Shall we make them consistent?

Reply:
yes, I'll fix it